Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -148,6 +148,14 @@ protected theorem IsClique.finsetMap {f : α ↪ β} {s : Finset α} (h : G.IsCl
     (G.map f).IsClique (s.map f) := by
   simpa
 
+/-- If a set of vertices `A` is a clique in subgraph of `G` induced by a superset of `A`,
+ it's embedding is a clique in `G`. -/
+theorem induce_isClique {S : Subgraph G} {F : Set α} {A : Set F} (c : (S.induce F).coe.IsClique A) :
```

Comment:
Depending on what you need it for it might also make sense to use `coeSpanning` instead along with an hypothesis of the form `(h : A ⊆ F)`. I think this is also fine, it just depends on your usecase. Also holds for the other lemma's.

Reply:
Do you mean something like this?
```
theorem spanning_isClique {S : Subgraph G} {A : Set S.verts} (c : S.spanningCoe.IsClique A) :
    G.IsClique (Subtype.val '' A) := by
```
I could also add those, if that's a use case that's expected to come up. The induced subgraph way perfect for my situation :)

Reply:
I meant something along these lines (but yours is also a valid alternative):

```
theorem induce_isClique {S : Subgraph G} {F A : Set α} (h : A ⊆ F)  (c : (S.induce F).spanningCoe.IsClique A)  : 
    G.IsClique A := by sorry
``

In any case, if you don't need either of these I'd personally say to just leave it at this until we do, but I'm happy to be overruled by someone with a better big picture overview:)